app/frontend/packs/nationalities-component.js

```diff
@@ -30,31 +30,33 @@ const nationalitiesComponent = () => {
   hideSection(thirdSelectEl, thirdFormLabel);
 
   function addRemoveLink(labelEl, selectEl) {
+    const parentEl = labelEl.parentElement;
+
     const removeLink = document.createElement("a");
     removeLink.innerHTML = "Remove";
     removeLink.classList.add("govuk-link", "app-nationality__remove-link");
 
     // This has to be a link and not a button as the govuk-link class requires an
-    // href to apply  it's styling
+    // href to apply it’s styling
```

Comment:
Apostrophe not req'd

Reply:
Ha! Blinded by the horror of an seeing ‘unsexy’ quote, completely neglected to see that it was unneeded 😆